Added AnalogThrottle plugin

MatheusMK3 opened the pull request.

austinvaness commented:
You seem to have some extra stuff in your repo that will be included in the plugin. MDK and Program.cs. I dont think you meant to include those did you?

austinvaness commented:
Actually, this project doesnt compile at all

MatheusMK3 commented:
Thanks for pointing out! I've removed the dependency on MDK and also removed the `Program.cs` file as it's not needed. Should be able to compile properly now.
